apps/jobs/serializers.py

```diff
@@ -56,3 +59,35 @@ def get_participant_team_name(self, obj):
 
     def get_leaderboard_schema(self, obj):
         return obj.leaderboard.schema
+
+
+class ViewSubmissionSerializer(serializers.ModelSerializer):
+
+    participant_team = serializers.SerializerMethodField()
+    challenge_phase = serializers.SerializerMethodField()
+    created_by = serializers.SerializerMethodField()
+    participant_team_members_email_id = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Submission
+        fields = ('id', 'participant_team', 'challenge_phase', 'created_by', 'status', 'is_public',
+                  'submission_number', 'submitted_at', 'execution_time', 'input_file', 'stdout_file',
+                  'stderr_file', 'submission_result_file', 'submission_metadata_file', 'participant_team_members_email_id',)
+
+    def get_participant_team(self, obj):
+        return obj.participant_team.team_name
+
+    def get_challenge_phase(self, obj):
+        return obj.challenge_phase.name
+
+    def get_created_by(self, obj):
+        return obj.created_by.username
+
+    def get_participant_team_members_email_id(self, obj):
```

Comment:
Since the function generally returns more than one email id, can we rename the function to `get_participant_team_members_email_ids`.